Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/internal/Corefinement/intersection_nodes.h

```diff
@@ -284,7 +285,15 @@ class Intersection_nodes<TriangleMesh,VertexPointMap,true,false>
   {
     put(vpm, vd, exact_to_double(enodes[i]));
     if (&tm1==&tm)
-      tm1_vertices[i] = vd;
+    {
+      if (  tm1_vertices[i] == GT::null_vertex() )
+      {
+        tm1_vertices[i] = vd;
+        return;
+      }
+      if (doing_autorefinement)
+        tm2_vertices[i]==vd;
+    }
```

Comment:
There is a warning in the testsuite... a useful one!

```
include/CGAL/Polygon_mesh_processing/internal/Corefinement/intersection_nodes.h:296:24: warning: equality comparison result unused [-Wunused-comparison]
        tm2_vertices[i]==vd;
        ~~~~~~~~~~~~~~~^~~~
```
https://cgal.geometryfactory.com/CGAL/testsuite/CGAL-4.13-Ic-59/Polyhedron_Demo/TestReport_cgaltester_x86-64_Darwin-13.0_Apple-clang-5.0_Release.gz